### ISSUE TITLE
adding the possibility to set / choose the linetype and possibly also the colors of the lines in plot_recruitment

### ISSUE THREAD

**Issue by nicoleb7** [SwissClinicalTrialOrganisation/secuTrialR#219]
**Is your feature request related to a problem? Please describe.**
When using plot_recruitment i would like to decide the linetype (not only dashed) and maybe even the colors of the lines

**Describe the solution you'd like**
plot_recruitment( x,
return_data = FALSE, 
show_centres = TRUE, 
cex = 1,
rm_regex = "",
linetype = (string of line types)
linecolor = (string of line colors)
)
**Describe alternatives you've considered**
do it all manually instead of using plot_recruitment
[inclusion_line_colors_and_types_free.pdf](https://github.com/SwissClinicalTrialOrganisation/secuTrialR/files/5317565/inclusion_line_colors_and_types_free.pdf)

**Additional context**

**Comment by aghaynes** [comment]
@PatrickRWright, can you look at this? It shouldn't be too difficult I wouldn't have thought...

**Comment by PatrickRWright** [comment]
Yes, I can do it when I have some free time. I asked @nicoleb7 to track the issue so I don't forget.

**Comment by aghaynes** [comment]
Super, thanks.
